.github/workflows/release.yaml

```diff
@@ -0,0 +1,61 @@
+---
+name: "Release"
+on:  # yamllint disable-line rule:truthy
+  push:
+    tags:
+      - "*"
+permissions:
+  contents: "write"
+  packages: "write"
+env:
+  GO_VERSION: "~1.22"
```

Comment:
I recommend removing this, so when the default version is updated in `setup-go`, all components using it get the latest go version